docs/UserGuide.md

```diff
@@ -775,6 +785,56 @@ Picture Example:
 | ----------- | ----------- |
 | ![november](images/CalendarPictures/Calendar_November.png) | ![october](images/CalendarPictures/Calendar_October.png) |
 
-## FAQ
+---
+
+
+### Command Format Summary Tables
```

Comment:
I think should push the table summary to the top so its easier for reference at any point in time. (not as instinctive to scroll all the way down, since the users might not know its there at all)

Reply:
oki will do